GraphsV2/Table/index.js

```diff
@@ -0,0 +1,1010 @@
+        infoBoxRow,
+        infoBoxColumn,
+        infoBoxData,
+        infoBoxScript,
+    }) => {
+        setShowInfoBox(true);
+        setInfoBoxRow(infoBoxRow);
+        setInfoBoxColumn(infoBoxColumn);
+        setInfoBoxData(infoBoxData);
+        setInfoBoxScript(infoBoxScript);
+    }
+
+    const onInfoBoxCloseHandler = () => setShowInfoBox(false);
+
+    const renderInfoBox = () => {
+        const {
+            Script,
+        } = props;
+
+        return (
+            showInfoBox &&
+            <InfoBox
+                onInfoBoxClose={onInfoBoxCloseHandler}
+            >
+                <Script
+                    row={infoBoxRow}
+                    key={infoBoxColumn}
+                    value={infoBoxData}
+                    script={infoBoxScript}
+                />
+            </InfoBox>
+        )
+    }
+
+    const renderConfirmationDialog = () => {
+        const actions = [
+            <FlatButton
+                label="Stay on Current Page"
+                labelStyle={style.button.labelStyle}
+                primary={true}
+                onClick={() => setShowConfirmationPopup(false)}
+            />,
+            <FlatButton
+                label="Continue"
+                labelStyle={style.button.labelStyle}
+                primary={true}
+                onClick={() => updateTableStatus({ currentPage: 1, selectedRow: {}, event: events.REFRESH }, props.updateScroll)}
+            />,
+        ];
+
+        return (
+            showConfirmationPopup &&
+            <React.Fragment>
+                <Dialog
+                    title="Unable to fetch"
+                    actions={actions}
+                    modal={true}
+                    contentClassName='dialogBody'
+                    open={true}
+                >
+                    Due to inactivity, we are not able to process the next page. Please press "Continue", to reload the data from first page.
+                </Dialog>
+            </React.Fragment>
+        );
+    }
+
+    useEffect(() => {
+        if(firstRender) {
+            firstRender = false;
+            let removedColumn = objectPath.has(props.scrollData, `removedColumn`) ? objectPath.get(props.scrollData, `removedColumn`) : uniq(removedColumns);
+            if (removedColumn.length <= 0) {
+                const columnsCurr = props.properties.columns || [];
+                const removeColumn = [];
+                columnsCurr.filter((col, key) => {
+                    if (col.display != undefined) {
+                        removeColumn.push("" + key);
+                    }
+                });
+                updateTableStatus({
+                    [`removedColumn`]: removeColumn,
+                    event: events.REMOVED_COLUMNS
+                }, props.updateScroll);
+            }
+        }
+        initiate(props); 
+        checkFontsize();
+        if (contextMenu) {
+            openContextMenu();
+        }
+        return () => {
```

Comment:
so this should executed on unmount. It's very hard to follow. Can you please create a function for this. Call it cleanup?